en/docs/learn/jwt-grant.md

```diff
@@ -0,0 +1,384 @@
+
+You have successfully added the identity provider. Next, you will configure the application as a service provider.
+
+### Service provider 
+
+To add the application as a service provider in WSO2 Identity Server:
+
+1.	On the **Main** menu, click **Identity > Service Providers > Add**.
+
+	<img src="../../assets/img/tutorials/add-sp-menu-item.jpeg" alt="Add New Service Provider menu-item" width="200"/>
+
+2. 	Enter a name for the service provider in the **Service Provider Name** text box and click **Register**.
+
+	<img src="../../assets/img/tutorials/add-sp-screen.jpeg" alt="Add New Service Provider screen" width="700"/> 		
+
+
+3. 	In the **Inbound Authentication Configuration** section, click **OAuth/OpenID Connect Configuration > Configure**.
+
+	<img src="../../assets/img/tutorials/oauth-openid-connect-configure-option.png" alt="OAuth OpenID Connect Configure option" width="700"/>
+
+4.	Enter the following configurations and click **Add**. 
+	
+	<a name="step3d"></a>
+
+	<img src="../../assets/img/tutorials/reg-new-app-screen.png" alt="Register New Application scree" width="700"/>
+
+	<table>
+		<thead>
+			<tr>
+				<th>Field</th>
+				<th>Description</th>
+				<th>Sample Value</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td>Callaback URL</td>
+				<td>This is the service provider's URL to which the access token will be sent. </td>
+				<td><code>http://localhost:8080/playground2/oauth2client</code>
+				</td>
+			</tr>
+			<tr>
+				<td>Enable Audience Restriction</td>
+				<td>This is to define whether the token will be sent to specific entities or not. <br>If this is not selected, the `client key` of the service provider will be sent in the `Aud` element of the JWT.</td>
+				<td>Selected</td>
+			</tr>
+			<tr>
+				<td>Audience</td>
+				<td>
+					<p>This is to define audiences for **this service provider** to which the token will be intended to.</p>
+					<div class="admonition tip">
+					<p class="admonition-title">Tip</p>
+					<p>To define audiences for **all the service providers** that are registered in WSO2 Identity Server globally:</p>
+					<ol>
+						<li>Open the `deployment.toml` file in the `<IS_HOME>/repository/conf/` directory.</li>
+						<li>Add the audiences using the following configuration. 
+						```toml
+						[oauth.oidc.id_token]
+						audiences=[] 
+						```
+						</li>
+						<li>Restart WSO2 Identityt Server and sign in to the Management Console.</li>
+					</ol>
+					</div>
+				</td>
+				<td>`https://localhost:9443/oauth2/token`</td>
+			</tr>
+		</tbody>
+	</table>		 
+
+	Note that the `client key` and `client secret` are generated. 
+
+	<img src="../../assets/img/tutorials/oidc-clientkey-clientsecret 2.png" alt="Client Key and Client Secret" width="700"/> 	
+
+	!!! note
+
+		While configuring the JWT grant type, the **iat** validating time period can also be configured in the `identity.xml` file in the `<IS_HOME>/repository/conf` as shown below. The default value is 30 minutes. 
+		
```

Comment:
not in the deployment.toml?

Reply:
The corresponding TOML configs are yet to be defined.